app/controllers/submitted_proposals_controller.rb

```diff
@@ -183,8 +183,11 @@ def post_to_editflow
 
     query_edit_flow = EditFlowService.new(@proposal).query
 
+    attach_supplementry_files
+
     response = RestClient.post ENV['EDITFLOW_API_URL'],
-                               { query: query_edit_flow, fileMain: File.open(@pdf_path) },
+                               { query: query_edit_flow, fileMain: File.open(@pdf_path),
+                                 fileMain: File.open(@supplementry_files_path) },
```

Comment:
Need to change role name (fileMain) for supplementary files, editflow not accepting any name for supplementary files, it should be asked from editflow team what we set name for 'multipartFormName' and attached supplementary files.